### PR TITLE
Add delphinet to metadata file, and create new conf file for delphi

### PR DESCRIPTION
**Pull request by PrathamRawat**

### CODE REVIEW

**Review by piotrkosecki — APPROVED** [review]

**Review by leenpaws — APPROVED** [review]

File: conseil-api/src/main/resources/metadata/tezos.delphinet.conf [hunk 1]
```diff
@@ -0,0 +1,1396 @@
...
+            "tz1PPUo28B8BroqmVCMMNDudG4ShA2bzicrU": "Tezos Korea"
+            "tz1b3SaPHFSw51r92ARcV5mGyYbSSsdFd5Gz": "Tezos MX"
+            "tz1Kf25fX1VdmYGSEzwFy1wNmkbSEZ2V83sY": "Tezos Seoul"
+            "tz1RAzdkp1x5ZDqms4ZUrSdfJuUYGH9JTPK2": "Tezos Spanish (Legacy)"
+            "tz1hAYfexyzPGG6RhZZMpDvAHifubsbb6kgn": "Tezos Suisse"
+            "tz1VYQpZvjVhv1CdcENuCNWJQXu1TWBJ8KTD": "Tezos Tokyo"
+            "tz1R6Ej25VSerE3MkSoEEeBjKHCDTFbpKuSX": "TezosSEAsia"
+            "tz1Lh9jeLSWDHYy8AshvG2dpNQseDaHg7cms": "TezosSpanish"
+            "tz1PesW5khQNhy4revu2ETvMtWPtuVyH2XkZ": "Tz Dutch"
+            "tz1Pwgj6j55akKCyvTwwr9X4np1RskSXpQY4": "Validators.com"
```
Comment (ivanopagano) [inline conseil-api/src/main/resources/metadata/tezos.delphinet.conf:976]: Question: are these kept as-is on delphi too? Or could they change?

File: conseil-api/src/main/resources/metadata/tezos.delphinet.conf [hunk 2]
```diff
@@ -0,0 +1,1396 @@
...
+            entity: "accounts"
+            key: "account_id"
+          }
+          cache-config {
+            cached: true,
+            min-match-length: 4,
+            max-result-size: 100
+          }
+          value-map: {
+            "tz1VxS7ff4YnZRs8b4mMP4WaMVpoQjuo1rjf": "Cryptonomic"
```
Comment (ivanopagano) [inline conseil-api/src/main/resources/metadata/tezos.delphinet.conf:851]: Question same as for bakers_history: does it remains the same as in carthage?

**Review by ivanopagano — APPROVED** [review]
Looks ok.

I raised a couple questions out of the review

File: conseil-api/src/main/resources/metadata/tezos.delphinet.conf (hunk 1 above)
Comment (PrathamRawat) [reply conseil-api/src/main/resources/metadata/tezos.delphinet.conf:976]: Delphi is a testnet so they will be different. Does that mean we should remove them from the PR?

Comment (ivanopagano) [reply conseil-api/src/main/resources/metadata/tezos.delphinet.conf:976]: maybe @anonymoussprocket or @vishakh knows better what to do

Comment (vishakh) [reply conseil-api/src/main/resources/metadata/tezos.delphinet.conf:976]: We should remove them from the PR.

File: conseil-api/src/main/resources/metadata/tezos.delphinet.conf (hunk 2 above)
Comment (vishakh) [reply conseil-api/src/main/resources/metadata/tezos.delphinet.conf:851]: It does not. We should remove such entries.

File: conseil-api/src/main/resources/metadata/tezos.delphinet.conf (hunk 1 above)
Comment (PrathamRawat) [reply conseil-api/src/main/resources/metadata/tezos.delphinet.conf]: They have been removed

**Comment by piotrkosecki** [comment]
one more thing I remembered, please add this line to the `Dockerfile` below other configs:
```
ADD ./conseil-api/src/main/resources/metadata/tezos.delphinet.conf /root/tezos.delphinet.conf
```

**Review by vishakh — APPROVED** [review]

**Pull request merged**